src/app/zap-templates/zcl/data-model/chip/device-energy-management-cluster.xml

```diff
@@ -0,0 +1,196 @@
+    <item fieldId="1" name="Value" type="int32s"/>
+    <item fieldId="2" name="DecimalPoints" type="int8u"/>
+    <item fieldId="3" name="Currency" type="int16u" max="999" optional="true"/>
+  </struct>
+  <struct name="PowerAdjustStruct" apiMaturity="provisional">
+    <cluster code="0x0098"/>
+    <item fieldId="0" name="MinPower" type="int64s"/>
+    <item fieldId="1" name="MaxPower" type="int64s"/>
+    <item fieldId="2" name="MinDuration" type="elapsed_s"/>
+    <item fieldId="3" name="MaxDuration" type="elapsed_s"/>
+  </struct>
+  <struct name="ForecastStruct" apiMaturity="provisional">
+    <cluster code="0x0098"/>
+    <item fieldId="0" name="ForecastId" type="int16u"/>
+    <item fieldId="1" name="ActiveSlotNumber" type="int16u" max="0xFFFE" isNullable="true"/>
+    <item fieldId="2" name="StartTime" type="epoch_s"/>
+    <item fieldId="3" name="EndTime" type="epoch_s"/>
+    <item fieldId="4" name="EarliestStartTime" type="epoch_s" max="0xFFFFFFFE" isNullable="true" optional="true"/>
+    <item fieldId="5" name="LatestEndTime" type="epoch_s" optional="true"/>
+    <item fieldId="6" name="IsPauseable" type="boolean"/>
+    <item fieldId="7" name="Slots" array="true" type="SlotStruct"/>
+  </struct>
+  <struct name="SlotStruct" apiMaturity="provisional">
+    <cluster code="0x0098"/>
+    <item fieldId="0" name="MinDuration" type="elapsed_s"/>
+    <item fieldId="1" name="MaxDuration" type="elapsed_s"/>
+    <item fieldId="2" name="DefaultDuration" type="elapsed_s"/>
+    <item fieldId="3" name="ElapsedSlotTime" type="elapsed_s"/>
+    <item fieldId="4" name="RemainingSlotTime" type="elapsed_s"/>
+    <item fieldId="5" name="SlotIsPauseable" type="boolean"/>
+    <item fieldId="6" name="MinPauseDuration" type="elapsed_s"/>
+    <item fieldId="7" name="MaxPauseDuration" type="elapsed_s"/>
+    <item fieldId="8" name="ManufacturerESAState" type="int16u" optional="true"/>
+    <item fieldId="9" name="NominalPower" type="int64s" optional="true"/>
+    <item fieldId="10" name="MinPower" type="int64s" optional="true"/>
+    <item fieldId="11" name="MaxPower" type="int64s" optional="true"/>
+    <item fieldId="12" name="NominalEnergy" type="int64s" optional="true"/>
+    <item fieldId="13" name="Costs" array="true" type="CostStruct" optional="true"/>
+    <item fieldId="14" name="MinPowerAdjustment" type="int64s" optional="true"/>
+    <item fieldId="15" name="MaxPowerAdjustment" type="int64s" optional="true"/>
+    <item fieldId="16" name="MinDurationAdjustment" type="elapsed_s" optional="true"/>
+    <item fieldId="17" name="MaxDurationAdjustment" type="elapsed_s" optional="true"/>
+  </struct>
+  <struct name="SlotAdjustmentStruct" apiMaturity="provisional">
+    <cluster code="0x0098"/>
+    <item fieldId="0" name="SlotIndex" type="int8u"/>
+    <item fieldId="1" name="NominalPower" type="int64s"/>
+    <item fieldId="2" name="Duration" type="elapsed_s"/>
+  </struct>
+  <struct name="ConstraintsStruct" apiMaturity="provisional">
```

Comment:
```suggestion
  <struct name="PowerLimitsStruct" apiMaturity="provisional">
```

Reply:
See note above about this being autogen'd from the in flight Spec PR.